sdk/ai/ai-projects/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/ai-projects",
-  "version": "1.0.0-beta.10",
+  "version": "1.0.0",
```

Comment:
@azure-rest/ai-inference is in beta, we shouldn't depend on a beta in a stable version.